scripts/pre_commit_linter.py

```diff
@@ -1312,6 +1336,84 @@ def _check_js_component_name_and_count(self):
             print ''
             return summary_messages
 
+    def check_for_mandatory_pattern_in_file(
```

Comment:
Why are we sending in `failed` as a parameter when the function returns `failed` itself?

Reply:
Actually the `failed` boolean is initialized in `_check_mandatory_patterns` and it is to be used in `check_for_mandatory_pattern_in_file` and it cannot be initialized in `check_for_mandatory_pattern_in_file` as it would refresh its value on every run which is not desired. Please let me know if there is any better way to do this.

Reply:
Okay, can you please make this function no-public (add an underscore before the function name)?